snapshot/store/store.go

```diff
@@ -50,7 +50,7 @@ type StoreRead interface {
 	Exists(name string) (bool, error)
 
 	// Open the bundle for streaming read. It is the caller's responsibility to call Close().
-	OpenForRead(name string) (io.ReadCloser, error)
+	OpenForRead(name string) (io.ReadCloser, *TTLValue, error)
```

Comment:
@dgassaway Thoughts on returning a struct with both io readcloser, and a ttlvalue?

Reply:
I'm fine either way. In a way the struct makes more sense. If we go that route Write() should use it as well.

Reply:
I was thinking about keeping write as is, to preserve backwards compatibility. Thoughts on that?

Reply:
We don't need to worry about backwards compat there, since this is just a code interface change, i.e. not changing a networked interface

Reply:
kk, done.